releasenotes/notes/0.22/add-barrier-label-8e677979cb37461e.yaml

```
@@ -0,0 +1,19 @@
+---
+features:
+  - |
+    Added a ``label`` parameter to the :class:`.Barrier` class's constructor
+    and the :meth:`~.QuantumCircuit.barrier` method which allows a user to
+    assign a label to an instance of the :class:`~.Barrier` directive. For
+    visualizations generated with :func:`~circuit_drawer` or
```

Does this need a leading `.` for sphinx to find the function?
```suggestion
    visualizations generated with :func:`~.circuit_drawer` or
```